### PR TITLE
Add error pages

### PR DESCRIPTION
**Pull request by thewilkybarkid**

### CODE REVIEW

**Comment by thewilkybarkid** [comment]
Needs more tests, but will have to merge before that for a demo.

File: vendor-extra/LiberoPageBundle/src/Resources/config/services.xml [hunk 1]
```diff
@@ -84,6 +92,30 @@
...
+        <service id="Libero\LiberoPageBundle\EventListener\Error\ServiceUnavailableErrorListener">
+            <argument type="service" id="assets.packages"/>
+            <argument type="service" id="translator"/>
+            <tag name="kernel.event_listener" event="libero.page.error.build" method="onBuildError"/>
+        </service>
+
+        <service id="Libero\LiberoPageBundle\EventListener\Error\ClientErrorListener">
+            <argument type="service" id="assets.packages"/>
+            <argument type="service" id="translator"/>
+            <tag name="kernel.event_listener" event="libero.page.error.build" method="onBuildError" priority="-100"/>
```
Comment (giorgiosironi) [inline vendor-extra/LiberoPageBundle/src/Resources/config/services.xml]: the priority of these last two seems low enough not to catch 404/503

File: vendor-extra/LiberoPageBundle/src/Resources/translations/messages+intl-icu.en.xlf [hunk 2]
```diff
@@ -28,6 +28,38 @@
                 <source>References</source>
             </trans-unit>
 
+            <trans-unit id="6" resname="libero.page.error.server_error.heading">
+                <source>Kabooooooom!</source>
```
Comment (giorgiosironi) [inline vendor-extra/LiberoPageBundle/src/Resources/translations/messages+intl-icu.en.xlf]: I wonder what process lead to defining exactly 7 `o` as the kaboom length

File: tests/WebTest.php [hunk 3]
```diff
@@ -154,4 +154,19 @@ public function idProvider() : iterable
         yield 'ID foo' => ['foo'];
         yield 'ID bar' => ['bar'];
     }
+
+    /**
+     * @test
+     */
+    public function it_shows_errors() : void
+    {
+        $client = self::createClient(['debug' => false]);
```
Comment (giorgiosironi) [inline tests/WebTest.php]: necessity to set `debug` to `false` prevents local testing right? I see the Symfony exception page:
Screenshot_2019-07-09 No route found for GET cdscds (404 Not Found)

Comment (thewilkybarkid) [reply tests/WebTest.php]: You can test by going to `/_error/404` etc.

Comment (thewilkybarkid) [reply tests/WebTest.php]: Or just use `docker-compose.yaml`.

Comment (giorgiosironi) [reply tests/WebTest.php]: The first works well for me.

**Pull request merged**